testnets/pockettestnet/chain.json

```diff
@@ -63,20 +61,21 @@
   },
   "images": [
     {
-      "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/testnets/pocket/images/pokt-icon.svg"
+      "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/testnets/pocket/images/pokt-icon.png"
```

Comment:
Please remove the images from the /testnets/pockettestnet/images/ directory (and the images directory itself). We should instead refer to the images in the mainnet directory to avoid uploading duplicate images.